azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/AvailabilitySetsImpl.java

```diff
@@ -96,7 +99,7 @@ public void delete(String groupName, String name) throws Exception {
         this.client.delete(groupName, name);
     }
 
-    /** Fluent model create helpers **/
+    /** Fluent model create helpers. **/
```

Comment:
Don’t worry about it, once this is merged I’ll fix all javadocs as part of resolving conflicts.

On Jun 8, 2016, at 6:00 PM, Anu Thomas Chandy <notifications@github.com<mailto:notifications@github.com>> wrote:


In azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/AvailabilitySetsImpl.java<https://github.com/Azure/azure-sdk-for-java/pull/781#discussion_r66367852>:

> @@ -96,7 +99,7 @@ public void delete(String groupName, String name) throws Exception {
>          this.client.delete(groupName, name);
>      }
>
> -    /** Fluent model create helpers **/
> +    /** Fluent model create helpers. **/


Ok, will fix them

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-sdk-for-java/pull/781/files/9e83f8d8a2029688f43d4b03a8471d4b4ec90dd5#r66367852>, or mute the thread<https://github.com/notifications/unsubscribe/ACNoPTgGMNOWTZpYCdX3NWyzu3TyDbdGks5qJ2WQgaJpZM4Ixfap>.